packages/editor-ui/src/components/DeleteUserModal.test.ts

```diff
@@ -0,0 +1,143 @@
+import { createComponentRenderer } from '@/__tests__/render';
+import DeleteUserModal from './DeleteUserModal.vue';
+import { createTestingPinia } from '@pinia/testing';
+import { getDropdownItems } from '@/__tests__/utils';
+import { createProjectListItem } from '@/__tests__/data/projects';
+import { createUser } from '@/__tests__/data/users';
+
+import { DELETE_USER_MODAL_KEY } from '@/constants';
+import { ProjectTypes } from '@/types/projects.types';
+import userEvent from '@testing-library/user-event';
+import { useUsersStore } from '@/stores/users.store';
+
+const ModalStub = {
+	template: `
+		<div>
+			<slot name="header" />
+			<slot name="title" />
+			<slot name="content" />
+			<slot name="footer" />
+		</div>
+	`,
+};
+
+const loggedInUser = createUser();
+const invitedUser = createUser({ firstName: undefined });
+const user = createUser();
+
+const initialState = {
+	ui: {
+		modalsById: {
+			[DELETE_USER_MODAL_KEY]: {
+				open: true,
+			},
+		},
+		modalStack: [DELETE_USER_MODAL_KEY],
+	},
+	projects: {
```

Comment:
```suggestion
	[STORES.PROJECTS]: {
```
Possibly missing from the constants, please add.